modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -609,6 +612,35 @@ def deserialize_model(model_data, model_class, path_to_item, check_type,
         instance = used_model_class(**kw_args)
     return instance
 
+recursive_discriminators_cache = {}
```

Comment:
Can you describe the structure of the dict that we are storing here?
What are the keys what are the values?